bin/opds2_import_monitor

```diff
@@ -11,6 +11,9 @@ from core.scripts import OPDSImportScript
 from core.model import ExternalIntegration
 from core.opds2_import import OPDS2Importer, OPDS2ImportMonitor
 
+# NOTE: We need to import it explicitly to initialize MirrorUploader.IMPLEMENTATION_REGISTRY
+from core import s3
```

Comment:
Re: import order - Can this be imported before`from core.scripts import OPDSImportScript`?